frontend/finance-tracker/src/components/ui/date-time-picker.tsx

```
@@ -0,0 +1,94 @@
+                    </Button>
+                </PopoverTrigger>
+                <PopoverContent className="w-auto p-0" align="start">
+                    <Calendar
+                        mode="single"
+                        selected={date}
+                        onSelect={(selectedDate) => {
+                            if (selectedDate) {
+                                const [hours, minutes] = selectedTime
+                                    ? selectedTime.split(":")
+                                    : ["0", "0"];
+                                selectedDate.setHours(parseInt(hours, 10));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)